src/main/java/frc/robot/subsystems/Revolver.java

```diff
@@ -58,7 +59,8 @@ public void resetUnjamTimer() {
 
   @Override
   public void periodic() {
-    if (m_revolver.get() > 0.0 && m_revolver.getEncoder().getVelocity() < Constants.Revolver.VELOCITY_THRESHOLD) {
+    if (m_setSpeed > 0.0
```

Comment:
Use `m_revolver.get()` once you have changed `m_revolver` to a `WPI_TalonFX`

Reply:
Resolved in commit 4b623d0